src/lightning/fabric/accelerators/mps.py

```diff
@@ -70,7 +71,8 @@ def auto_device_count() -> int:
     @lru_cache(1)
     def is_available() -> bool:
         """MPS is only available on a machine with the ARM-based Apple Silicon processors."""
-        return torch.backends.mps.is_available() and platform.processor() in ("arm", "arm64")
+        mps_disabled = os.getenv("DISABLE_MPS", "0") == "1"
```

Comment:
You could have done this by adding this condition to the RunIf decorator so that the tests get skipped

Reply:
It's what I did first but there are many tests that don't use the RunIf decorator and rely on auto-accelerator detection. For these tests, I want to just have `MPSAccelerator.is_available()` return False so they run on CPU instead.